src/components/quote/Quote.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
+import './Quote.css';
+
+function Quote() {
+  const [quote, setquote] = useState({});
+
+  useEffect(() => {
+    fetch('https://api.quotable.io/random')
+      .then((response) => response.json())
+      .then((data) => {
+        setquote(data);
+      });
+  }, []);
+
+  return (
+    <div className="quote">
+      {quote.content ? (
+        <blockquote>
+          &ldquo;
+          {quote.content}
+          &rdquo; -
+          {' '}
+          {quote.author}
+        </blockquote>
+      ) : (
+        <p>Loading ....</p>
+      )}
+    </div>
+  );
```

Comment:
- [ ] I like the loading displayed while waiting for the quotes to load, good job :+1: